qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -112,6 +110,25 @@
             driver_test_name_balloon = "balloon"
             iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
             read_rng_cmd = "WIN_UTILS:\\random_%PROCESSOR_ARCHITECTURE%.exe"
+            x86_64:
+                # virtio-mem is not supported on 32-bit systems
+                # and viomem is supported from 1.9.40-0
+                boot_with_viomem = yes
+                driver_test_name_viomem = "viomem"
+                maxmem_mem = 80G
+                mem_devs = 'vmem0'
+                vm_memdev_model_vmem0 = "virtio-mem"
+                size_mem_vmem0 = 8G
+                requested-size_memory_vmem0 = 1G
+                memdev_memory_vmem0 = "mem-vmem0"
+                requested-size_test_vmem0 = "4G 8G 0"
+                test_drivers += ' viomem'
+                driver_test_name_viomem = "viomem"
+            i386:
+                # mainly to cover win10.i386 test without viomem
+                boot_with_viomem = no
+            Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1, Host_RHEL.m9.u2, Host_RHEL.m9.u3, Host_RHEL.m9.u4:
```

Comment:
I am considering how about setting `no Host_RHEL.m7 ...`, as it was not supported in earlier hosts.

Reply:
You mean: no Host_RHEL.m7, Host_RHEL.m8 Host_RHEL.m9.u0 Host_RHEL.m9.u1 Host_RHEL.m9.u2 Host_RHEL.m9.u3 Host_RHEL.m9.u4 idirectly? if so we won't filter this case if we run it on a rhel930 host without viomem device.

Reply:
Yes, it makes sense.

Reply:
> Yes, it makes sense.

but we use the  host version as the case condition, it mean we still expect to filter the case if run on a rhel930 host  without viomem, feel free to contact me if my understanding is wrong.